docs/eventing/broker/kafka-broker/README.md

```diff
@@ -227,6 +227,40 @@ kubectl create secret --namespace <namespace> generic <my_secret> \
 !!! note
     `ca.crt` can be omitted to fallback to use system's root CA set.
 
+## Consumer Offsets Commit Interval
+
+Kafka consumers keep track of the last successfully sent events by committing offsets.
+
+For performance reasons, it's not recommended committing offsets every time an event is successfully sent to a
+subscriber.
+
+Knative Kafka Broker commits the offset every `auto.commit.interval.ms` milliseconds.
+
+`auto.commit.interval.ms` can be changed by changing the `config-kafka-broker-data-plane` `ConfigMap`
+in the `knative-eventing` namespace by changing the parameter `auto.commit.interval.ms` as follows:
+
+```yaml
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-kafka-broker-data-plane
+  namespace: knative-eventing
+data:
+  # Some configurations omitted ...
+  config-kafka-broker-consumer.properties: |
+    # Some configurations omitted ...
+
+    # Commit the offset every 5000 millisecods (5 seconds)
+    auto.commit.interval.ms=5000
+```
+
+!!! note
+    While Knative Kafka Broker guarantees at least once delivery, hence your applications may
+    receive duplicate events, the higher the commit interval is the higher is the probability
+    of receiving duplicate events since when a Consumer restarts, it starts again from last
+    committed offsets.
+
```

Comment:
```suggestion
## Consumer offsets commit interval

Kafka consumers keep track of the most recent, successfully sent events by committing offsets.

The Knative Kafka Broker commits offsets every `auto.commit.interval.ms` milliseconds.

!!! note
    To prevent negative impacts to performance, it is not recommended to commit offsets every time an event is successfully sent to a subscriber.

This interval can be modified in the `config-kafka-broker-data-plane` ConfigMap in the `knative-eventing` namespace, by modifying the `auto.commit.interval.ms` parameter as follows:

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: config-kafka-broker-data-plane
  namespace: knative-eventing
data:
  # Some configurations omitted ...
  config-kafka-broker-consumer.properties: |
    # Some configurations omitted ...

    # Commit the offset every 5000 millisecods (5 seconds)
    auto.commit.interval.ms=5000
```
!!! note
    Knative Kafka Broker guarantees at least once delivery, which means that your applications may receive duplicate events. A higher commit interval means that there is a higher probability of receiving duplicate events, because when a Consumer restarts, it restarts from the last committed offset.
```

Reply:
For some reason the suggestion is formatting weird, so you don't need to commit this - you can see my suggestions for changes and feel free to use ones that make sense though

Reply:
Thank you, that's much better!